content/de/blog/v2-release/index.md

```diff
@@ -0,0 +1,43 @@
+
+## Umfangreiche Anfragenhistorie
+
+Wenn Du unsere Webseite regelmäßig nutzt, kennst Du vermutlich unsere „Meine Anfragen“-Seite. Wenn Du es nicht in den {{< link slug="/privacy-controls" text="Datenschutzeinstellungen" >}} ausgeschaltet hast, speichern wir alle Anfragen, die Du verschickst, in Deinem Browser und geben Dir darüber eine Übersicht. Bisher war das nur eine unübersichtliche Tabelle, die wir schon seit Jahren verbessern und als richtiges Anfragenverwaltungstool ausbauen wollten. In unserem neuen Release erstellen wir jetzt einen Vorgang für jede Deine Anfragen, der alle Nachrichten enthält, die etwas mit ihr zu tun haben, einschließlich Mahnungen und Beschwerden. Du kannst Informationen über Nachrichten, die Du von der Verantwortlichen erhältst, hinzufügen, sodass Du alles an einem Ort gesammelt hast, falls Du dann eine Beschwerde senden möchtest. Wir speichern den Status all Deiner Anfragen, damit Du ihren aktuellen Stand verfolgen kannst, und erinnern Dich auf allen Seiten daran, falls es überfällige Anfragen gibt, für die Du eine Mahnung verschicken solltest.
+
+{{< img name="my-requests" alt="Screenshot der „Meine Anfragen“-Seite. Unter dem Erklärungstext stehen mehrere Vorgänge mit verschiedenen Status, wie etwa „wartet auf Antwort“, „überfällig“ und „abgeschlossen“. Ein Vorgang ist aufgeklappt und enthält zwei Nachrichten, eine von der Nutzer_in und eine des angefragten Unternehmens." caption="Die neue Anfragenhistorie auf der „Meine Anfragen“-Seite." >}}
+
+## Beschwerdegenerator und Anfragereaktionen
+
+Die neuen “Meine Anfragen” Seite kommt erst mit einem weiteren Feature so richtig zur Geltung, auf das wir besonders stolz sind: Basierend auf Deiner Anfragenhistorie geben wir Dir die Möglichkeit zu “reagieren”. Wenn Du Dich dazu entscheidest, helfen wir Dir einzuschätzen, welche Fehler eine Verantwortliche möglicherweise gemacht hat, indem wir Dir ein paar Fragen stellen. Daraus generieren wir dann eine Mahnung oder sogar eine richtige Beschwerde. Auch wenn wir nochmal deutlich machen wollen, dass wir keine Anwälte sind und keine Rechtsberatung für spezifische Fälle anbieten können, hoffen wir trotzdem, dass diese Feature es Dir leichter macht, Dich bei den Aufsichtsbehörden zu beschwerden. Du bist mit Deiner Überforderung von Unternehmensantworten nicht mehr allein.
```

Comment:
```suggestion
Die neuen „Meine Anfragen“-Seite kommt erst mit einem weiteren Feature so richtig zur Geltung, auf das wir besonders stolz sind: Basierend auf Deiner Anfragenhistorie geben wir Dir die Möglichkeit zu „reagieren“. Wenn Du Dich dazu entscheidest, helfen wir Dir einzuschätzen, welche Fehler eine Verantwortliche möglicherweise gemacht hat, indem wir Dir ein paar Fragen stellen. Daraus generieren wir dann eine Mahnung oder sogar eine richtige Beschwerde. Auch wenn wir nochmal deutlich machen wollen, dass wir keine Anwälte sind und keine Rechtsberatung für spezifische Fälle anbieten können, hoffen wir trotzdem, dass diese Feature es Dir leichter macht, Dich bei den Aufsichtsbehörden zu beschweren. Du bist mit Deiner Überforderung von Unternehmensantworten nicht mehr allein.
```

Reply:
I already didn't particularly like the English version but I doubt many people are going to understand "Du bist mit Deiner Überforderung von Unternehmensantworten nicht mehr allein.".